### PR TITLE
Enable custom 404 pages for external web containers

### PR DESCRIPTION
**Pull request by mlitcher**
Removed external container check. This gives external web containers the ability to use CustomErrorMessages.

### CODE REVIEW

**Comment by tipsy** [comment]
Since this behavior is very old (pre custom error pages), I want to check with @perwendel before merging it.

**Comment by perwendel** [comment]
I'm trying to figure out why `&& !externalContainer` is there to begin with!

**Comment by mlitcher** [comment]
Since this is the only usage of the `externalContainer` flag, would it make sense to also remove it completely from this class, including the constructor? If you agree, I can update this PR.

**Comment by perwendel** [comment]
Update:
I've been on vacation for 6 weeks that now is over so work on the project will start again. I will start review PRs ASAP.

File: src/main/java/spark/Service.java [hunk 1]
```diff
@@ -532,6 +532,16 @@ private void initializeRouteMatcher() {
         }
     }
 
+    /**
+     * @return The approximate number of currently active threads in the embedded Jetty server
+     */
+    public synchronized int activeThreadCount() {
```
Comment (perwendel) [inline src/main/java/spark/Service.java]: rename to `activeThreads()`

File: src/main/java/spark/Spark.java [hunk 2]
```diff
@@ -1197,4 +1197,11 @@ public static ModelAndView modelAndView(Object model, String viewName) {
         return new ModelAndView(model, viewName);
     }
 
+    /**
+     * @return The approximate number of currently active threads in the embedded Jetty server
+     */
+    public static int activeThreadCount() {
```
Comment (perwendel) [inline src/main/java/spark/Spark.java]: rename to `activeThreads()`

**Comment by mlitcher** [comment]
Just looking back at this, it looks like I inadvertently pushed unrelated commits into this pull request. I'm thinking I should re-do this pull with only my change?

**Comment by tipsy** [comment]
> I'm thinking I should re-do this pull with only my change?

Please do.

**Pull request closed without merging**